contracts/Comptroller/Diamond/facets/SetterFacet.sol

```diff
@@ -789,8 +789,12 @@ contract SetterFacet is ISetterFacet, FacetBase {
 
         require(newLiquidationIncentiveMantissa >= mantissaOne, "incentive < 1e18");
 
-        // Emit event with old incentive, new incentive
-        emit NewLiquidationIncentive(poolId, market.liquidationIncentiveMantissa, newLiquidationIncentiveMantissa);
+        emit NewLiquidationIncentive(
```

Comment:
`NewMarketLiquidationIncentive`? As the incentives are now on a per-market basis, it will be more informative.